src/frontend-pwa/src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+
+export type SliderProps = {
+  min: number;
+  max:number;
+  onChange: (value: number) => void;
+}
+
+export default function Slider({
+  min,
+  max,
+  onChange,
+}:SliderProps) {
+  const [value, setValue] = useState(min);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const newValue = parseInt(event.target.value, 10);
+    setValue(newValue);
+    onChange(newValue);
+  };
+
+  return (
+    <div>
+      <input
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/slider_role
Needs to take in feasible accessibility labels